packages/table/preview/localStore.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ * Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+ * of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+ * and https://github.com/palantir/blueprint/blob/master/PATENTS
+ *
+ * Demonstrates sample usage of the table component.
+ */
+
+import { defaults } from "lodash";
+
+export class LocalStore<T> {
+    private storage: Storage;
+
+    constructor(private key: string, session = false){
+        this.storage = session ? sessionStorage : localStorage;
+    }
+
+    public getWithDefaults(defs?: T): T | {} {
```

Comment:
Would prefer to call this `defaults`. `defs` means "definitions" to me.

Reply:
name already taken by the lodash method, but i'll come up w somethign

Reply:
Ah, boo.